core/crop/ingress_adv.py

```diff
@@ -7,8 +7,8 @@
 import os
 import pandas as pd
 
-# from crop.db import create_database
-from crop.constants import (
+# from core.crop.db import create_database
+from core.crop.constants import (
```

Comment:
Instead of adding **core.**, could you please adjust your python path since we have restructured the repository.